include/pacemaker.h

```diff
@@ -12,6 +12,7 @@
 
 #  include <glib.h>
 #  include <libxml/tree.h>
+#  include <crm/cib/cib_types.h>
```

Comment:
Unrelated to this, but it made me notice: lib/pacemaker/Makefile.am and lib/libpacemaker.pc.in don't link to libstonithd or libcrmservice but should

Reply:
Added PR #2628 for this.